<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HistogramBox</class>
 <widget class="QWidget" name="HistogramBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>88</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QwtPlot" name="qwtPlot"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="layout_options">
        <item>
         <widget class="QToolButton" name="btn_Options">
          <property name="text">
           <string>options</string>
          </property>
          <property name="icon">
           <iconset resource="../../../icons/histogram.qrc">
            <normaloff>:/histogram/advancedsettings.png</normaloff>:/histogram/advancedsettings.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btn_Load">
          <property name="text">
           <string>open</string>
          </property>
          <property name="icon">
           <iconset resource="../../../icons/histogram.qrc">
            <normaloff>:/histogram/fileopen.png</normaloff>:/histogram/fileopen.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btn_Save">
          <property name="text">
           <string>save</string>
          </property>
          <property name="icon">
           <iconset resource="../../../icons/histogram.qrc">
            <normaloff>:/histogram/filesave.png</normaloff>:/histogram/filesave.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btn_Vertical">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../../../icons/histogram.qrc">
            <normaloff>:/histogram/object-flip-vertical.png</normaloff>:/histogram/object-flip-vertical.png</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btn_Horizontal">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../../../icons/histogram.qrc">
            <normaloff>:/histogram/object-flip-horizontal.png</normaloff>:/histogram/object-flip-horizontal.png</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btn_Statistic">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>S</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="btn_Clear">
          <property name="text">
           <string>clear</string>
          </property>
          <property name="icon">
           <iconset resource="../../../icons/histogram.qrc">
            <normaloff>:/histogram/edit-clear.png</normaloff>:/histogram/edit-clear.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../icons/histogram.qrc"/>
 </resources>
 <connections/>
</ui>
